### ISSUE TITLE
Reduce size of pyinstaller package

### ISSUE THREAD

**Issue by samtygier-stfc** [mantidproject/mantidimaging#1658]
Currently pyinstaller produces a package that is  2.5 GB (zipped 1.1GB) on windows, and even larger on linux (due to shared library duplications).

It should be possible to reduces the package size by removing some unused parts.

**Comment by JackEAllen** [comment]
@samtygier-stfc Do you think we can reduce the size further very easily or should we consider closing this issue for now?

**Comment by samtygier-stfc** [comment]
Once MI moves the Qt6 it can drop the Qt5 packages (currently I think it still has both, because matplotlib has a dependency on Qt6).

There was also talk at some point of the cudatoolkit packages being split into multiple modules, so would be worth checking.

It might also be possible to exclude some individual files from packages.

**Comment by JackEAllen** [comment]
yes, I saw #1842 for splitting up Cuda. I'l; bring both into 3.2.0 Milestone as we should explore while moving to Qt6 anyway
